### ISSUE TITLE
Auto play doesn't working when use  clicked on navigation arrow

### ISSUE THREAD

**Issue by smartHasmukh** [Useful-Software-Solutions-Ltd/angular2-useful-swiper#25]
Auto play doesn't working when use  clicked on navigation arrow or pagination

**TypeScript Code**

`vericalSlider: Object = {
    paginationClickable: true,
    nextButton: '.swiper-button-next',
    prevButton: '.swiper-button-prev',
    spaceBetween: 20,
    autoplay: true,
    speed: 2000,
    direction: 'vertical',
    touchEventsTarget: 'container',
    initialSlide: 0,
    autoplayDisableOnInteraction: true,
    autoplayStopOnLast: false,
    iOSEdgeSwipeDetection: false,
    iOSEdgeSwipeThreshold: 20,
  };`

**HTML Code**

`<swiper class="home-slider" [config]="vericalSlider">
              <div class="swiper-wrapper">
                <div class="swiper-slide">Latest trading value up by 2%</div>
                <div class="swiper-slide">Latest trading value up by 4%</div>
                <div class="swiper-slide">Latest trading value up by 5%</div>
              </div>
              
              <div class="swiper-pagination"></div>
              <div class="swiper-button-next"></div>
              <div class="swiper-button-prev"></div>
            </swiper>`

**Comment by JayChase** [comment]
Hi. please use the demo Plunk http://embed.plnkr.co/qM4jHG/ to recreate any issues.

**Comment by smartHasmukh** [comment]
@JayChase  I have made [Demo](https://plnkr.co/edit/2JDFSYDPeGL0GMwOOaRh?p=preview) for plnkr.

I have assign `loadImages()` methods on `ngOnInit()` so all image will be loaded on when plnkr run and I have added 2 properties `autoplay:true,speed:3000` on config object. right now images swiper(move) with 3 seconds but when I have to click on `swiper-button-next` or `swiper-button-prev` arrow then swiper(move) will stop.

Autoplay doesn't work after clicked on next or prev arrows
